cosmoz-moment.html

```diff
@@ -12,10 +31,10 @@
 		window.Cosmoz = window.Cosmoz || {};
 
 		var localSharedMoment = moment(),
```

Comment:
it was removed from cosmoz-moment component properties but still exist in behavior properties